guides/development/hack_on_the_trento.md

```diff
@@ -1,82 +1,93 @@
-# Hack on the trento
-
-- [Requirements](#requirements)
-- [Install dependencies](#install-dependencies)
-- [Development environment](#development-environment)
-- [Setup trento](#setup-trento)
-- [Start trento in the repl](#start-trento-in-the-repl)
-- [Environment Variables](#environment-variables)
-- [Scenario loading with photofinish](#scenario-loading-with-photofinish)
+# Hack on the Trento Web
 
 ## Requirements
 
-In order to run the trento web, you need
+In order to run the Trento Web application, the following software must be installed:
 
 1. [Elixir](https://elixir-lang.org/)
-2. [Erlang OTP](https://www.erlang.org/)
-3. [Docker](https://docs.docker.com/get-docker/)
-4. [Docker Compose](https://docs.docker.com/compose/install/)
+2. [Docker](https://docs.docker.com/get-docker/)
+3. [Docker Compose](https://docs.docker.com/compose/install/)
 
 ## Development environment
 
-A `docker-compose` development environment is provided.
+The Trento project provides a docker-compose development environment that is used to start a Postgres database and a Grafana instance for storage and monitoring. To start the development environment, navigate to the root directory of the Trento project and run the following command:
 
 ```
 docker-compose up -d
 ```
 
-It will start a **postgres** database and a **grafana** instance, for storage and monitoring.
-## Install dependencies
+## Setup Trento
+
+Before starting Trento Web, some initial setup tasks, like installing dependencies and creating the database, are required.
+Execute following command:
 
 ```
-mix deps.get
+mix setup
 ```
 
-## Setup trento
+## Connect Trento Web with [Wanda](https://github.com/trento-project/wanda)
+
+By default, Wanda can be accessed on port 4001. To connect Trento Web to Wanda, create a .env file in the root path of Trento Web.
```

Comment:
Not really _in the root path of Trento Web_. That's more like the root path of the client app (aka react app, js app or whatever-you-wanna-call-it app :smile: )